api/src/api/index.ts

```diff
@@ -11,6 +12,7 @@ router.use('/users', users);
 router.use('/pitch', pitch);
 router.use('/auth', auth);
 router.use('/resources', resources);
+router.use('/interest', interests);
```

Comment:
Should the endpoint be "interests" to fit with teams? I see "pitch" is also singular though...

```suggestion
router.use('/interests', interests);
```